docs/usage.md

```diff
@@ -3,130 +3,441 @@
+
+### Review Enriched Indicators
+
+- Verify the accuracy and relevance of the reputation information retrieved. Ensure that the data reflects the most current assessment of the indicator.
+
+### Integrate Enriched Indicators into Workflow
+
+- Use the enriched reputation information to inform your security analysis and response actions. Incorporate the data into your workflow to enhance decision-making and threat management.
+
+>[!Tip]
+>Regularly fetch and review the latest reputation information for indicators to stay informed about potential threats. Up-to-date data helps maintain effective security measures.
+
+### Conclusion
+
+Enriching indicators is a critical task for understanding threat levels within FortiSOAR. By following the outlined steps and leveraging the FortiAI assistant’s capabilities, users can effectively retrieve and utilize the latest reputation information to support their security operations.
+
+>[!Note]
+>Ensure that the sources used for fetching reputation information are reliable and up-to-date to maintain the accuracy of the data.
+
+## Escalate To Incident
+
+Escalating an alert to an incident within the FortiAI assistant involves promoting an alert to a higher level of response. This functionality is essential for managing critical alerts that require more detailed investigation and response actions.
+
+### Use Case Example: Escalating an Alert to an Incident
+
+A common use case is escalating an alert that indicates a significant security threat, which needs to be tracked and managed as an incident. For example, an alert about a potential data breach may be escalated to an incident for comprehensive investigation and remediation.
+
+### Escalating Alerts Using FortiAI Assistant
+
+1. **Initiate Escalation Action**:
+   - Issue a command to escalate the alert to an incident. For example:
+     > *Promote alert with ID [ALERT-ID] to an incident*
+   - The assistant triggers a playbook that asks for information like incident name and severity before escalating the alert to the incident.
+
+2. **Provide Alert Details**:
+   - Specify the details of the alert that needs to be escalated. For example:
+     > *Escalate alert [ALERT-ID] with subject [ALERT-SUBJECT] to incident*
+   - The assistant processes the escalation request and generates the incident accordingly.
+
+3. **Review Incident Creation**:
+   - Verify that the incident has been created and that all relevant details from the alert have been transferred. Ensure that the incident record accurately reflects the information from the alert.
+
+### Review Escalated Incidents
+
+- Confirm that the escalation process has been completed successfully and that the new incident record contains all necessary information. Review the incident to ensure it is properly documented and ready for further action.
+
+### Integrate Escalated Incidents into Workflow
```

Comment:
same as above. Do we really need these review sections and multiple texts?

Reply:
Consistency